tools/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/google/addlicense v1.1.1
 	github.com/open-policy-agent/conftest v0.41.0
 	github.com/tektoncd/chains v0.15.1-0.20230406140737-50a40c019838
-	github.com/tektoncd/cli v0.3.1-0.20230419203848-832d14301e20
+	github.com/tektoncd/cli v0.30.1
```

Comment:
We want the version after 30.1 that includes https://github.com/tektoncd/cli/commit/832d14301e20ed8864e9067203904b52e438a898